src/date-picker/style.ts

```diff
@@ -50,16 +50,21 @@ kuc-date-picker[hidden] {
   display: none;
 }
 .kuc-date-picker__group input.kuc-base-date__input {
```

Comment:
```suggestion
.kuc-date-picker__group input[type=text].kuc-base-date__input {
```
If the component is in the group field, the color of text is black when input box is focusing.